packages/tenants-nodejs/src/plugin.ts

```diff
@@ -94,13 +113,8 @@ export const init = createPluginInitFunction<
               method: "get",
               verifySessionOptions: {
                 sessionRequired: true,
-                overrideGlobalClaimValidators: (globalValidators) => {
-                  if (!pluginConfig.requireNonPublicTenantAssociation) {
-                    return globalValidators;
-                  }
-
-                  return validateWithoutClaim(globalValidators, MultipleTenantsPresentClaim.key);
-                },
+                overrideGlobalClaimValidators: (globalValidators) =>
+                  validateWithoutClaims(globalValidators, [MultipleTenantsPresentClaim.key, "st-perm"]),
```

Comment:
Please make `validateWithoutClaims` return the lambda itself so the syntax becomes like:
```suggestion
                overrideGlobalClaimValidators: validateWithoutClaims([MultipleTenantsPresentClaim.key, "st-perm"]),
```